crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -1077,6 +1080,11 @@ pub async fn run(
             telemetry::configure(command, &mut base, child_event);
             Ok(0)
         }
+        Command::Scan {} => {
+            let base = CommandBase::new(cli_args.clone(), repo_root, version, ui);
+            scan::run(base).await;
+            Ok(0)
```

Comment:
I suspect we don't always want exit code 0. Maybe only if we don't find anything?

Reply:
Changed